hadoop-hdds/config/src/main/java/org/apache/hadoop/hdds/conf/TimeDurationUtil.java

```diff
@@ -50,6 +50,9 @@ public static long getTimeDurationHelper(String name, String vStr,
     if (null == vUnit) {
       LOG.warn("No unit for " + name + "(" + vStr + ") assuming " + unit);
       vUnit = ParsedTimeDuration.unitFor(unit);
+      if(null == vUnit){
```

Comment:
Nit:

```suggestion
      if (null == vUnit) {
```

Reply:
Small details overlooked,thanks for reminding~